src/cls/IPM/Lifecycle/Base.cls

```diff
@@ -1296,6 +1296,11 @@ Method %Publish(ByRef pParams) As %Status
 {
 	Set tSC = $$$OK
 	Try {
+		// Perform the "Package" phase to prepare the module for publishing
+		$$$ThrowOnError(..OnBeforePhase("Package", .pParams))
+		$$$ThrowOnError(..%Package(.pParams))
```

Comment:
So the way to do this, is to call `..Package` and allow DispatchMethod/DispatchClassMethod to call ExecutePhases() which will run packaging as expected. Can see how its done for `..Configure()` and `..MakeDeployed()` as examples (worth adding a comment though since it is confusing since there is no such method named in this class or the super classes.